project/api/models.py

```diff
@@ -135,6 +135,29 @@ def save(self, *args, **kwargs):  # pylint: disable=W0221
         super(CachedData, self).save(*args, **kwargs)
 
 
+class City(models.Model):
+    """
+    Stores a city represented by a point on the map
+    """
+
+    wikidata_id = models.PositiveIntegerField()  # Excluding the Q
+    label = models.TextField(max_length=50)
```

Comment:
50 symbols might not be enough.
I found an [article](https://www.worldatlas.com/articles/the-10-longest-place-names-in-the-world.html), according to this article longest name of the city is 85 symbols, and next is 56
I know, that two cities is not a big deal.